### ISSUE TITLE
GUI is displayed black

### ISSUE THREAD

**Issue by SticcyGem** [ArtixAllMighty/PublicGuiAnnouncement#20]
So basically me and my friends love this mod and played with it on an SMP but some people are having issues with the rendering. We all noticed that everyone with AMD integrated graphics have rendering issues.

These Pictures we're taken by 3 different people with AMD integrated graphics.

![sample_1](https://user-images.githubusercontent.com/92389082/192034929-83936873-db08-46dc-b069-acf068ca9302.png)
![sample_3](https://user-images.githubusercontent.com/92389082/192034951-7a9dc909-450e-40e6-afbc-c9509f96f35f.png)
![sample_2](https://user-images.githubusercontent.com/92389082/192034962-f48fae32-ec1b-47c8-8d04-4f6fe54c2fd5.png)

Anyone else with a Dedicated GPU, and Intel integrated graphics have no problems.

**Comment by AbsolemJackdaw** [comment]
have you tried playing without optifine / sodium ? or any other mdos that mess with 'render optimization' ?

**Comment by SticcyGem** [comment]
Yeah we can try, I'll update the situation once everyone affected is online in a few hours.

**Comment by SticcyGem** [comment]
So some very important notes:
We are all playing on 1.19
We are using your mod in version: 1.19-4.0.0.0

We removed sodium and everything still display black
![sample_4](https://user-images.githubusercontent.com/92389082/192157028-715a5fc2-01c8-4872-bf6a-7363a418881b.png)

Then we tried using version 1.19.2-4.3.0.0 on a 1.19 server but failed to start up.

Then we tried using version 1.19.2-4.3.0.0 on a 1.19,2 world without sodium/optifine
![sample_5](https://user-images.githubusercontent.com/92389082/192157137-34f9e69b-6a9d-432c-922e-f968b0a2c59b.png)
![sample_6](https://user-images.githubusercontent.com/92389082/192157149-51233775-722f-4345-9572-8f46884a4c69.png)

Everything is still renders black on AMD integrated graphics.

**Comment by SticcyGem** [comment]
Would be really nice if there is a patch for 1.19 version too since some of the mods are not updated for 1.19.2 yet.

**Comment by AbsolemJackdaw** [comment]
I have looked into the issue and I sadly cannot do anything about integrated graphics. I'm using no custom rendering code nor do I reference openGL to render the screens. 

I would like to suggest to not use vsync (which will give you more fps btw, on integrated graphics) and try to disable smooth lighting and fancy clouds, ... basically, run minecraft on its lowest setting, to make sure no interference between minecraft and your integrated graphics causes the issues. 

If those do not work, then I don't really know what I can do to help

**Comment by ouroborus** [comment]
From the images, it looks like the alpha channel is working fine.

**Comment by AbsolemJackdaw** [comment]
the lighting issue has been fixed in 1.21.1 and 1.21.10. 
ill see if i can make a quick patch for 1.19

**Comment by AbsolemJackdaw** [comment]
the envoironement for 1.19 is tricky to set back up with all the modenr changes. i'm going to pass on it for now. 
let me know if you still play on 1.19, you may reopen the ticket. 
for now i'll close it due to 1.19 being a stale/old version
